conf/messages

```diff
@@ -93,16 +93,16 @@ nisp.main.context.improve.list.payable.plural = You have {0} years on your recor
 nisp.main.context.improve.nogaps = You cannot improve your forecast any further. You''ll still need to pay contributions as these fund other state benefits and the NHS.
 nisp.main.context.improve.nogaps.line2 = When you reach State Pension age, {0}, you no longer have to pay National Insurance contributions.
 
-nisp.main.context.improve.para1.plural = You have {0} years on your National Insurance record which do not count towards your State Pension. You may improve your forecast by filling them.
-nisp.main.context.improve.para1.singular = You have {0} year on your National Insurance record which does not count towards your State Pension. You may improve your forecast by filling it.
+nisp.main.context.improve.para1.plural = You have years on your National Insurance record which do not count towards your State Pension. You may improve your forecast by filling them.
+nisp.main.context.improve.para1.singular = You have a year on your National Insurance record which does not count towards your State Pension. You may improve your forecast by filling it.
```

Comment:
@fred-jackson wouldn't this have a similar issue as before if the one gap occurs pre-2006? Is this okay?

Or maybe even for the plural if all the gaps are non-fillable?